backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -98,6 +98,22 @@ public void testGetTokensForShop_success() throws Exception {
                 inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().get(shopId).size());
     }
 
+    @Test
+    public void testGetShop_success() {
+        // Arrange.
+        UUID shopId = addShopToRepository();
+        
+        // Act.
+        Shop newShop = inMemoryVirtualQueueRepository.getShop(shopId);
+
+        // Assert.
+        assertEquals("Shop Owner Id ", SHOP_OWNER_ID, newShop.getShopOwnerId());
+        assertEquals("Shop Name ", SHOP_NAME, newShop.getShopName());
+        assertEquals("Shop Address ", SHOP_ADDRESS, newShop.getAddress());
+        assertEquals("Shop Phone Number ", PHONE_NUMBER, newShop.getPhoneNumber());
+        assertEquals("Shop Type ", SHOP_TYPE, newShop.getShopType());
+    }
+
     private UUID addShopToRepository() {
         Shop shop = new Shop(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
```

Comment:
Can we pass these 5 values as params to the  addShopToRepository method. That will make the tests more readable. Right now, we need to look into the logic of addShopToRepository to understand the assert statements and to understand from where are values SHOP_OWNER_ID, SHOP_NAME, etc in assert statements are coming from. If we pass these as params to addShopToRepository method, we'll make each test more independent and more readable.

Reply:
Done.